omero/sysadmins/unix/install-web/install-apache.txt

```diff
@@ -26,6 +35,27 @@ Install `Django 1.8`_ using package requirements file:
           :djangodoc:`how to install Django 1.8 <topics/install/#install-the-django-code>`
           or :djangodoc:`hot to upgrade Django to 1.8 <topics/install/#remove-any-old-versions-of-django>`.
 
+Additional settings can be configured by the following properties:
+
+- :property:`omero.web.application_server.max_requests` to 500
+
+- :property:`omero.web.wsgi.workers` (2 x NUM_CORES) + 1
+
+  .. note::
+      DO NOT scale the number of workers to the number of clients
+      you expect to have. OMERO.web should only need 4-12 worker
```

Comment:
`-` should be an en-dash (`–`) since this is a range (this applies to all use of ranges in the document).